server/runtime/src/main/resources/schema/infinispan-server-11.0.xsd

```diff
@@ -80,6 +82,146 @@
       </xs:simpleContent>
    </xs:complexType>
 
+   <xs:complexType name="data-sources">
+      <xs:sequence>
+         <xs:element type="tns:data-source" name="data-source" maxOccurs="unbounded" minOccurs="0"/>
+      </xs:sequence>
+   </xs:complexType>
+
+   <xs:complexType name="data-source">
+      <xs:all>
+         <xs:element type="tns:data-source-connection-factory" name="connection-factory">
+            <xs:annotation>
+               <xs:documentation><![CDATA[ Configuration for the connection factory ]]></xs:documentation>
```

Comment:
Why are the CDATA blocks required throughout when the content is just a String in most cases?

Reply:
I copied those from the WildFly agroal subsystem XSD :)